codeforamerica/vita-min
# Nj 27 add nj

## Link to pivotal/JIRA issue
https://github.com/newjersey/affordability-pm/issues/27
## Is PM acceptance required? (delete one)
- Yes?
## What was done?
Add NJ following NC and Warshington examples.
## How to test?
Added fake NJ directfile import xml, copied from NY examples. You can manually run through the process an import one to see it working.

Also added landing page controller test and return xml test.  Added NJ to schema file loader and state information service tests.
